slack-base/src/main/java/com/hubspot/slack/client/models/files/SlackFileType.java

```diff
@@ -5,14 +5,15 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
 
-public enum SlackFileTypes {
+public enum SlackFileType {
   TEXT("text"),
   GIF("gif"),
+  CSV("csv")
```

Comment:
can we also include a trailing `,`, just so the next addition doesn't touch this line in the dif?

Reply:
Yes, I  usually do that and I'm sorry I didn't